docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,13 +1,117 @@
-# Deploy to Fly.io with SQLite
+# Deploy Platformatic Applications to Fly.io
 
-:::note
+## Deploying a Platformatic Runtime Application 
+
+This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+
+
+### Dockerfile for Runtime Application
+
+Here is an example Dockerfile for a Platformatic Runtime application:
+
+```dockerfile
+FROM node:20-alpine
+
+ENV APP_HOME=/home/app/node/
+
+WORKDIR $APP_HOME
+
+COPY . .
```

Comment:
Can you verify if `node_modules` is copied? I think by default `.gitignore` is treated as `.dockerignore`, but I might not remembering correctly.

We should mention that users would need a `.dockerignore` file to avoid copying `node_modules`

Reply:
Yes, `.dockerignore` is treated as `.gitignore` and I've added that users need a dockerignore file to avoid copying node_modules

Reply:
Yes, .dockerignore is treated as `.gitignore` and I've added that users need a `.dockerignore` file to avoid copying `node_modules`